apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-examples/platform-radio-group-list-items-example.component.html

```diff
@@ -1,60 +1,102 @@
-<p>Reactive form Examples</p>
-<form [formGroup]="customForm" class="flex-form">
-    <p>Example 1: Radio Buttons Types with default value and error status</p>
-    <fieldset fd-fieldset>
+<h3>Use With Reactive Form</h3>
+
+<p>Platform Radio Buttons Created with given List of SelectItem Objects and have Validation Error</p>
+<fdp-form-group [multiLayout]="true" [formGroup]="form1">
+    <fdp-form-field [id]="'radiolo1'" zone="zLeft" rank="1">
         <fdp-radio-group
-            [name]="'radio1'"
-            [value]="'Option 2'"
+            [name]="'radiolo1'"
+            [value]="'spring'"
             [list]="items"
             [isInline]="false"
             formControlName="example1"
         >
         </fdp-radio-group>
-    </fieldset>
-    <p>Example 2: Radio Buttons Types in Compact contentDensity</p>
-    <fieldset fd-fieldset>
+    </fdp-form-field>
+</fdp-form-group>
+<p style="margin-left: 2rem;">Selected value: {{ form1.controls.example1?.value }}</p>
+<br />
+
+<p>Compact Platform Radio Button Created with given List of SelectItem Objects. Using FormGroup and FormControl.</p>
```

Comment:
Period after "Objects" is not necessary. 

Also — I'm not sure if this is a title or a description, but if it's a title: capitalize "Given," if it's not a title then please see comment below.

Reply:
It is a description.
corrected.